content/core-features/authentication.textile

```diff
@@ -211,6 +211,39 @@ ably.auth.requestToken({ clientId: 'client@example.com' }, function(err, tokenDe
 
 *Note:* The authenticity of the JWT *will not* be checked, due to Ably not having access to your SECRET key.
 
+h4(#jwt-claims). Using JWT for authenticated user claims
+
+It is possible for JWTs to contain authenticated claims for users that can be used to allow or disallow certain interactions in your channels.
+
+Messages can be annotated with trusted metadata copied from the client's authentication token by Ably servers. Clients are unable to directly publish messages with user claim metadata and claims contained within the authentication token are signed to prevent tampering. Claims can be scoped to individual channels or to namespaces of channels, see our "namespaces documentation":/general/channel-rules-namespaces/ for more information. The most specific user claim will be annotated to the message as part of the `extras` object.
+
+*Note:* This does not apply to presence or metadata messages.
+
+An example use case is when using "Message Interactions":/realtime/messages#message-interactions. You might want to use trusted claims to define 'moderator' users in a chat channel who have the ability to delete any sent messages. When the moderator sends an interaction to mark messages in a channel as deleted, your application should check that user's claims to verify they are a moderator for that channel before actioning their request.
+
+To set the trusted fields you need to include `ably.channel.` in your JWT authentication payload, for example:
```

Comment:
> include `ably.channel.`

Markup should use `@`. Also, should the period be in or out of the markup - almost looked like a wildcard was implied.